shared/actions/kbfs/index.js

```diff
@@ -1,39 +1,3 @@
 // @flow
-import * as KBFSGen from '../kbfs-gen'
-import * as Platform from './index.platform'
-import * as Saga from '../../util/saga'
-import * as RPCTypes from '../../constants/types/flow-types'
-import {isWindows} from '../../constants/platform'
-
-function* _listSaga(action: KBFSGen.ListPayload): Saga.SagaGenerator<any, any> {
-  try {
-    const result: ?RPCTypes.ListResult = yield Saga.call(RPCTypes.fsListRpcPromise, {
-      path: action.payload.path,
-    })
-
-    if (result) {
-      console.log('fs.List: ', result)
-      yield Saga.put(KBFSGen.createListed({result}))
-    }
-  } catch (error) {
-    yield Saga.put(KBFSGen.createListedError({error}))
-  }
-}
-
-function* kbfsSaga(): Saga.SagaGenerator<any, any> {
```

Comment:
this delegates it all to platform and native does nothing. instead lets just have the platform control the global saga